### PR TITLE
Feature/new build

### PR DESCRIPTION
**Pull request by gwelter**
New Build without create-react-app

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

**Comment by joshLong145** [comment]
@ntindle builders seem to hanging on this pr are our builders down?

**Review by ntindle — CHANGES_REQUESTED** [review]
Dont commit .DS_store

**Comment by ntindle** [comment]
Our builders don't "go down". It uses github actions. If the builder isn't running, it hasn't been triggered or GH is down

**Comment by ntindle** [comment]
Try resolving the conflicts and see if it will run

**Comment by joshLong145** [comment]
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: @sonr-io/nebula-react@0.1.0
npm ERR! Found: react@18.1.0
npm ERR! node_modules/react
npm ERR!   dev react@"18.1.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8.0 || ^17.0.0" from @storybook/react@6.4.22
npm ERR! node_modules/@storybook/react
npm ERR!   dev @storybook/react@"6.4.22" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/joshlong/.npm/eresolve-report.txt for a full report.
```
When I run `npm install` I get the following peer depedency conflictions, seems to be incompatibility with react 18 and storybook. perhaps we downgrade to 17 or 16

**Comment by joshLong145** [comment]
> ```
> npm ERR! code ERESOLVE
> npm ERR! ERESOLVE unable to resolve dependency tree
> npm ERR! 
> npm ERR! While resolving: @sonr-io/nebula-react@0.1.0
> npm ERR! Found: react@18.1.0
> npm ERR! node_modules/react
> npm ERR!   dev react@"18.1.0" from the root project
> npm ERR! 
> npm ERR! Could not resolve dependency:
> npm ERR! peer react@"^16.8.0 || ^17.0.0" from @storybook/react@6.4.22
> npm ERR! node_modules/@storybook/react
> npm ERR!   dev @storybook/react@"6.4.22" from the root project
> npm ERR! 
> npm ERR! Fix the upstream dependency conflict, or retry
> npm ERR! this command with --force, or --legacy-peer-deps
> npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
> npm ERR! 
> npm ERR! See /Users/joshlong/.npm/eresolve-report.txt for a full report.
> ```
> 
> When I run `npm install` I get the following peer depedency conflictions, seems to be incompatibility with react 18 and storybook. perhaps we downgrade to 17 or 16

Thinking about it, this is probably why the actions dont resolve, cant install the dependency tree and hangs up most likely.

File: package.json [hunk 1]
```diff
@@ -8,82 +8,62 @@
...
-    "start": "react-scripts start",
-    "build": "rollup --config",
-    "test": "jest --config ./jest.config.js",
-    "eject": "react-scripts eject",
-    "storybook": "start-storybook -p 6006 -s public",
-    "build-storybook": "build-storybook -s public",
-    "prepack": "npm run test"
+  "peerDependencies": {
+    "react": "^18.1.0",
+    "react-dom": "^18.1.0"
```
[the remark this answers is not in the document]
Comment (joshLong145) [reply package.json]: Looks like we cannot target major version 18 in our repo as of this time due to storybook being incompatible. we should downgrade back to our previous version for compatibility

[the remark this answers is not in the document]
Comment (gwelter) [reply package.json]: Downgraded to react 17.0.2 (@testing-library/react also since newer versions had react 18 as peer a dependency)

**Review by joshLong145 — APPROVED** [review]

**Review by ntindle — APPROVED** [review]

**Pull request merged**